synedrion/Cargo.toml

```diff
@@ -10,23 +10,23 @@ readme = "README.md"
 categories = ["cryptography", "no-std"]
 
 [dependencies]
-signature = { version = "2", default-features = false, features = ["alloc"] }
-k256 = { version = "0.13.2", default-features = false, features = ["ecdsa", "arithmetic"] }
-rand_core = { version = "0.6.4", default-features = false }
-sha2 = { version = "0.10", default-features = false }
-sha3 = { version = "0.10", default-features = false }
-digest = { version = "0.10", default-features = false, features = ["alloc"]}
+signature = { version = "2.3.0-pre.4", default-features = false, features = ["alloc"] }
+k256 = {git = "https://github.com/RustCrypto/elliptic-curves", default-features = false, features = ["ecdsa", "arithmetic"]}
+rand_core = { version = "0.6.4", default-features = false, features = ["getrandom"] }
+sha2 = { version = "0.11.0-pre.4", default-features = false }
+sha3 = { version = "0.11.0-pre.4", default-features = false }
+digest = { version = "0.11.0-pre.9", default-features = false, features = ["alloc"]}
 hex = { version = "0.4", default-features = false, features = ["alloc"] }
-base64 = { version = "0.21", default-features = false, features = ["alloc"] }
+base64 = { version = "0.22.1", default-features = false, features = ["alloc"] }
 hashing-serializer = { version = "0.1", default-features = false }
 secrecy = { version = "0.9.0-pre.0", default-features = false, features = ["serde"] }
 zeroize = { version = "1.8", default-features = false, features = ["alloc", "zeroize_derive"] }
-bip32 = { version = "0.5.2", default-features = false, features = ["alloc", "secp256k1"] }
+bip32 = { version = "0.5.2", default-features = false, features = ["alloc", "secp256k1", "k256"] }
 
 # Note: `alloc` is needed for `crytpto-bigint`'s dependency `serdect` to be able
 # to serialize Uints in human-readable formats.
-crypto-bigint = { version = "0.5.3", default-features = false, features = ["serde", "alloc"] }
-crypto-primes = { version = "0.5", default-features = false }
+crypto-bigint = { version = "0.6.0-rc.3", features = ["serde", "alloc", "rand_core"] }
```

Comment:
I think the `rand_core` here may be the thing that's breaking the WASM build in CI

Reply:
Why do we even need this feature outside of `dev-dependencies`? What part of the code uses it?